eng/resources.targets

```diff
@@ -15,20 +12,19 @@
   </PropertyGroup>
 
   <!-- Include files under StringResourcesPath by convention unless OmitResources is set. -->
-  <ItemGroup Condition="'$(StringResourcesPath)' != '' and '$(OmitResources)' != 'true'">
-    <EmbeddedResource Include="$(StringResourcesPath)">
-      <Visible>true</Visible>
-      <ManifestResourceName>$(StringResourcesName)</ManifestResourceName>
-      <GenerateSource>true</GenerateSource>
-      <ClassName>$(StringResourcesNamespace).$(StringResourcesClassName)</ClassName>
-    </EmbeddedResource>
-  </ItemGroup>
-
-  <!-- Include common SR helper when resources are included unless SkipCommonResourcesIncludes is set. -->
-  <ItemGroup Condition="Exists('$(StringResourcesPath)') and '$(OmitResources)' != 'true' and '$(SkipCommonResourcesIncludes)' == ''">
```

Comment:
The `SkipCommonResourcesIncludes` switch is dead-code as it isn't used anywhere anymore.